src/features/integration/components/TagList.tsx

```diff
@@ -0,0 +1,24 @@
+import {Box, Typography} from "@mui/material";
+import {Tag} from "./dnd/Tag";
+import {TaglistPopoverContent, tagList} from "../util/DefaultValues";
+import * as React from "react";
+import HelpPopover from "./popover/HelpPopover";
+
+const TagList: React.FunctionComponent<any> = (props) => {
```

Comment:
Is there a better name for Tag?

Reply:
separate task https://fintlabs.atlassian.net/browse/FST-123